wasm/src/wasm_builtins.rs

```diff
@@ -10,19 +10,31 @@ extern crate web_sys;
 use rustpython_vm::obj::objstr;
 use rustpython_vm::VirtualMachine;
 use rustpython_vm::pyobject::{ PyFuncArgs, PyResult };
-use web_sys::console;
+use wasm_bindgen::JsCast;
+use web_sys::{HtmlTextAreaElement, window};
+
+// The HTML id of the textarea element that act as our STDOUT
+const CONSOLE_ELEMENT_ID: &str = "console";
+
+fn print_to_html(text: &str) {
```

Comment:
Would it be possible to have this function in javascript and import it into rust with a wasm bindgen like macro? If so, we could to without the window, htmltextareaelement rust types. Or is this actually better, since this is statically typed rust code?

Reply:
Woops, now I read your original comment. Please ignore mine.